### DISCUSSION TITLE
How to handle cross-cutting/shared concerns in Federation (Summit 7/2020)

### DISCUSSION THREAD

**Discussion by jhampton** [apollographql/apollo-server#4432] [General]
# Context

In the Federation topic room, we discussed the desire to establish strong patterns around shared concerns:

* Value Types (non-entities)
* Scalars
* Enums

## Experience

We have shared experience here that could help others:

* Using shared libraries
* Using "layers" of schema or a non-planable set of schema
* What can you share about your experience?

## Make this an "Impossible State"

* What could we see in Federation that would reduce the risk of this shared concern?
* Are there extensions to Federation that could help?

**Comment by smyrick** [comment]
@jhampton It looks like this repo has access to the Github Discussions Beta. 

Should we move this over there? https://github.com/apollographql/apollo-server/discussions
